### PR TITLE
Add example qmake.

### PR DESCRIPTION
**Pull request by Djabx**
I've added a dummy qmake exemple.
Qt source come from: https://github.com/nesayus/gradle.qt

### CODE REVIEW

File: examples/qmake/qmake.pro [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+SOURCES += \
+    src/main/cpp/main.cpp \
+    src/main/cpp/MyQtObject.cpp
+
+# The following define makes your compiler emit warnings if you use
+# any feature of Qt which as been marked deprecated (the exact warnings
+# depend on your compiler). Please consult the documentation of the
+# deprecated API in order to know how to port your code away from it.
+DEFINES += QT_DEPRECATED_WARNINGS
```
Comment (awrichar) [inline examples/qmake/qmake.pro]: I think this block (and the block below) are unnecessary for the example. IMO better to keep the .pro file very simple to demonstrate integration with the Gradle plugin.

File: examples/qmake/qmake.pro [hunk 2]
```diff
@@ -0,0 +1,32 @@
+QT += core
+QT -= gui
+
+CONFIG += c++11
```
Comment (awrichar) [inline examples/qmake/qmake.pro]: You didn't use any C++11 constructs in the example code, did you? If not, you can remove this, to make sure the example compiles on as many systems as possible.

**Comment by awrichar** [comment]
Thank you for this! Can you correct the spelling in the commit message ("exemple" -> "example")? I also suggested a few simplifications in the .pro file, just to keep this example as generic as possible. If you can make these small changes, I'll be happy to integrate this.

**Comment by Djabx** [comment]
Done.

**Comment by awrichar** [comment]
Thanks!

**Pull request merged**